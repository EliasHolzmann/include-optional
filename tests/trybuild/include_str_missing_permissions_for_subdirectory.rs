fn main() {
    /* This does not fail only when executing `include_str`, but already when executing
     * `include_str_optional` â€“ we don't have permissions to look inside the `fd` directory, so
     * `Path::try_exists` returns `Error(_)`
     */
    include_optional::include_str_optional!("/proc/1/fd/doesnt_exist");
}