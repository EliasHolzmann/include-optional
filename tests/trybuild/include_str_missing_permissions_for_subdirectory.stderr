error: Couldn't access /proc/1/fd/doesnt_exist: Permission denied (os error 13)
 --> $DIR/include_str_missing_permissions_for_subdirectory.rs:6:45
  |
6 |     include_optional::include_str_optional!("/proc/1/fd/doesnt_exist");
  |                                             ^^^^^^^^^^^^^^^^^^^^^^^^^
